### ISSUE TITLE
There were errors detected during the run of the NEST test suite!

### ISSUE THREAD

**Issue by AlexStasik** [nest/nest-simulator#893]
NEST Testsuite Summary
----------------------
  NEST Executable: /Users/astasik/NEST/bin/nest
  SLI Executable : /Users/astasik/NEST/bin/sli
  Total number of tests: 959
     Passed: 945
     Skipped: 13 (7 PyNEST)
     Failed: 1 (1 PyNEST)

***
*** There were errors detected during the run of the NEST test suite!
***
*** Please report the problem at
***     https://github.com/nest/nest-simulator/issues
***
*** To help us diagnose the problem, please attach the archived content
*** of these directories to the issue:
***     - '/Users/astasik/NEST/bld/reports'
***     - '/var/folders/sb/tjqnvvps1mzb4k4x_fbj3shc0000gn/T//nest.2h8ZL'
***

make[3]: *** [CMakeFiles/installcheck] Error 1
make[2]: *** [CMakeFiles/installcheck.dir/all] Error 2
make[1]: *** [CMakeFiles/installcheck.dir/rule] Error 2
make: *** [installcheck] Error 2
[reports.zip](https://github.com/nest/nest-simulator/files/1745079/reports.zip)
[T.zip](https://github.com/nest/nest-simulator/files/1745085/T.zip)

**Comment by terhorstd** [comment]
There seems to be an issue with the new python handling the filter() function.
```
======================================================================
ERROR: test_synapse_deletion_one_to_one_no_sp (nest.tests.test_sp.test_disconnect.TestDisconnectSingle)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/astasik/NEST/lib/python3.6/site-packages/nest/tests/test_sp/test_disconnect.py", line 86, in test_synapse_deletion_one_to_one_no_sp
    assert len(conns) == 1
TypeError: object of type 'filter' has no len()
```

**Comment by heplesser** [comment]
@AlexStasik Thank you for reporting this issue! This has already been fixed in the NEST master branch (c37b022c94); the fix will be included in the next NEST release. It is only due to a minor Python 2/3 issue in the test and does not indicate any problem in NEST itself.
